### DISCUSSION TITLE
Add option to ignore other encrypted zfs datasets

### DISCUSSION THREAD

**Discussion by sotiris-bos** [zbm-dev/zfsbootmenu#570] [Feature Requests]
I am using ZBM on my desktop, laptop and server.

My desktop and laptop get backed up to the server via syncoid. I am backing up everything, including zfs and zpool attributes to another pool on the server.

ZFSBootMenu on the server tries to unlock every encrypted root dataset, including the ones that have been backed up to the server by the desktop and laptop.

It would be great to have a way to make ZBM ignore those datasets. Preferably file-based instead of zfs/zpool attribute.

**Comment by zdykstra** [comment]
Use zbm.prefer to restrict the pool import process to your primary pool.

**Comment by sotiris-bos** [comment]
> Use zbm.prefer to restrict the pool import process to your primary pool.

My bad, I had set zbm.prefer to the dataset instead of the pool, thank you!
